src/comments.ts

```diff
@@ -53,6 +53,23 @@ export async function updateComment(
   return updatedComment.data
 }
 
+export async function deleteComment(
```

Comment:
What if we minimize the old comments rather than deleting them? There's a capability in the new GraphQL API that allows this. https://docs.github.com/en/graphql/reference/mutations#minimizecomment

This fn might turn into something like this:

```typescript
export async function minimizeComment(
   octokit: InstanceType<typeof GitHub>,
   subjectId: number,
   classifier? = 'OUTDATED'
 ): Promise<void> {
  await octokit.graphql(`
    mutation($input: MinimizeCommentInput!) { 
      minimizeComment(input: $input) {
        clientMutationId
      }
    }`,
    {input: {subjectId, classifier}}
  )
 }
```

Reply:
Actually it's not the purpose of the proposition (and it's not interesting me in the way I'm using my modification: posting last CI results from GA to current PR ;)). But would be great to also have the possibility of doing it!